ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -245,25 +245,46 @@ function getValues(pendingApproval, t, actions, history, data) {
         ],
       },
 
-      {
-        element: 'Typography',
-        key: 'title',
-        children: originIsMetaMask
-          ? t('wantToAddThisNetwork')
-          : t('addEthereumChainConfirmationTitle'),
-        props: {
-          variant: TypographyVariant.H3,
-          align: 'center',
-          fontWeight: 'bold',
-          boxProps: {
-            margin: [0, 0, 4],
+      process.env.CHAIN_PERMISSIONS
+        ? {
+            element: 'Typography',
+            key: 'title',
+            children: originIsMetaMask
+              ? t('wantToAddThisNetwork')
+              : t('addNetworkConfirmationTitle', [
+                  pendingApproval.requestData.chainName,
+                ]),
+            props: {
+              variant: TypographyVariant.H3,
+              align: 'center',
+              fontWeight: 'bold',
+              boxProps: {
+                margin: [0, 0, 4],
+              },
+            },
+          }
+        : {
+            element: 'Typography',
+            key: 'title',
+            children: originIsMetaMask
+              ? t('wantToAddThisNetwork')
+              : t('addEthereumChainConfirmationTitle'),
+            props: {
+              variant: TypographyVariant.H3,
+              align: 'center',
+              fontWeight: 'bold',
+              boxProps: {
+                margin: [0, 0, 4],
+              },
+            },
           },
-        },
-      },
+
       {
         element: 'Typography',
         key: 'description',
-        children: t('addEthereumChainConfirmationDescription'),
+        children: process.env.CHAIN_PERMISSIONS
```

Comment:
As we reference this a few times, could we define a top level constant to avoid the duplicated `process.env`?

Reply:
I think keeping `process.env.CHAIN_PERMISSIONS` here should be fine for the scenario when we will be planning to remove it. Doing global search in the code base will be helpful to find where it's used and we can remove it. wdyt?